src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -9,13 +9,13 @@
 5
 
 >> Value(ParseJSON("true"))
```

Comment:
@jorisdg  - can you confirm the cases here?

Reply:
Looks good, I'm not seeing Boolean ( ParseJSON("5") ) though. Considering some languages define anything other than zero to be "true" we should explicitly define our stance here.

Reply:
I asked Joris to look at the tests in Teams.